sdk/servicebus/azure-servicebus/tests/test_sb_client.py

```diff
@@ -125,4 +125,15 @@ def test_sb_client_incorrect_queue_conn_str(self, servicebus_queue_authorization
         with client:
             with pytest.raises(ServiceBusError):
                 with client.get_queue_sender(wrong_queue.name) as sender:
-                    sender.send_messages(Message("test"))
\ No newline at end of file
+                    sender.send_messages(Message("test"))
+
+    def test_sb_client_wrong_user_agent(self):
```

Comment:
async version of this?

Reply:
tests not needed as we decided not to do validation for now until when needed.